src/main/java/com/projetogerenciamentocurso/gerenciamentocurso/controller/AlunoController.java

```diff
@@ -0,0 +1,65 @@
+package com.projetogerenciamentocurso.gerenciamentocurso.controller;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.projetogerenciamentocurso.gerenciamentocurso.dto.AlunoDto;
+import com.projetogerenciamentocurso.gerenciamentocurso.dtoresposta.AlunoDtoResposta;
+import com.projetogerenciamentocurso.gerenciamentocurso.exceptions.AlunoException;
+import com.projetogerenciamentocurso.gerenciamentocurso.service.AlunoService;
+
+import lombok.AllArgsConstructor;
+
+@RestController
+@RequestMapping(path = AlunoController.PATH)
+@AllArgsConstructor
+public class AlunoController {
+
+	public static final String PATH = "/aluno";
+
+	
+	private final AlunoService alunoService;
+	
+	@PostMapping
+	public ResponseEntity<AlunoDtoResposta> salvarAluno(@RequestBody AlunoDto aluno)throws AlunoException {

```

Comment:
Pela exceção ser uma RuntimeException não é necessário colocar um throws. Fazer isso em todos os outros Controllers
```suggestion
	public ResponseEntity<AlunoDtoResposta> salvarAluno(@RequestBody AlunoDto aluno){
```

Reply:
feito